src/types/runtime/gravity/Video.ts

```diff
@@ -0,0 +1,31 @@
+import { GraphQLInt, GraphQLObjectType, GraphQLString } from "graphql"
+import { Number, Record, String } from "runtypes"
+import { ResolverContext } from "../../graphql"
+
+export const VideoType = new GraphQLObjectType<any, ResolverContext>({
+  name: "Video",
+  description: "An object containing video metadata",
+  fields: {
+    src: {
+      description: "The url of the video",
+      type: GraphQLString,
+      resolve: ({ src }) => src,
+    },
+    height: {
+      description: "The height of the video",
+      type: GraphQLInt,
+      resolve: ({ height }) => height,
+    },
+    width: {
+      description: "The width of the video",
+      type: GraphQLInt,
```

Comment:
I'd make all these fields non-null since we're—currently—guaranteed to have this data. `new GraphQLNonNull(GraphQLInt)`